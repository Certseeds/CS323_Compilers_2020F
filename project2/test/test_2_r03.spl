int test_2_r03()
{
  int ttt = 3;
  float iii = 0.3;
  float kkk = 1.2;
  float ttt = iii * kkk;
  return 0;
}
