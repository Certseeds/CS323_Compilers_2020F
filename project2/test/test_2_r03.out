Error type 3 at Line 6: redefine variable: ttt
